specs/electra/validator.md

```diff
@@ -13,6 +13,7 @@
     - [Attester slashings](#attester-slashings)
     - [Attestations](#attestations)
     - [Deposits](#deposits)
+    - [Execution payload](#execution-payload)
 - [Attesting](#attesting)
   - [Construct attestation](#construct-attestation)
 - [Attestation aggregation](#attestation-aggregation)
```

Comment:
Are there any changes with respect to `is_aggregator`? I was reading the eip and it sounded like now it'll be done by weight